packages/account/src/Components/api-token/api-token-context.ts

```diff
@@ -1,6 +1,12 @@
 import * as React from 'react';
 import { TApiContext } from 'Types';
 
-const ApiTokenContext = React.createContext<TApiContext>({});
+const ApiTokenContext = React.createContext<TApiContext>({
```

Comment:
What is the purpose of this?

Reply:
This is to provide an Initial value towards the ApiContext based on the types given in `TApiContext`